source/scripting_v3/GTA/Entities/Peds/Ped.cs

```diff
@@ -962,9 +962,9 @@ private int GetEventTypeIndexForB1737OrOlder(EventType eventType)
 		#region Euphoria & Ragdoll
 
 		/// <inheritdoc cref="Ragdoll(int, RagdollType, bool)"/>
-		public void Ragdoll(int duration = -1, RagdollType ragdollType = RagdollType.Relax)
```

Comment:
This signature is shipped in the stable release v3.6.0. Although the other one is not shipped in it, we cannot change the return type of this signature.